sourced/ml/tests/test_language_selector.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+from pyspark.sql.functions import lit
+
+from sourced.ml.tests.models import PARQUET_DIR, SIVA_DIR
+from sourced.ml.transformers import LanguageSelector, Collector, Ignition, HeadFiles
+from sourced.ml.transformers.basic import create_parquet_loader, create_engine
+
+
+class LoaderTest(unittest.TestCase):
+    def test_parquet(self):
+        languages1 = ["Python", "Java"]
+        languages2 = ["Java"]
+
+        engine = create_engine("test", SIVA_DIR)
+        res = Ignition(engine) \
+            .link(HeadFiles()) \
```

Comment:
Looking at these frequent `link()` calls I have got a crazy offtopic idea: how about

```python
class Transformer:
    def __rshift__(self, other):
        return self.link(other)
```

and then

```python
res = Ignition(engine) >> HeadFiles() >> LanguageSelector() >> ...
```

Thus if our train is too long to fit on a single line we can easily discard all the `\`:

```python
res = (Ignition(engine) >>
       HeadFiles() >>
       LanguageSelector() >>
        ...
```

Reply:
@r0mainK @warenlg @EgorBu what's your opinion? Or shall we use `__add__` instead?

Reply:
I like the look of it personally, however don't really like:
```
res = (Ignition(engine) >>
       HeadFiles() >>
       LanguageSelector() \
       .execute()
```

Reply:
it looks nice to me also +1

Reply:
@r0mainK We can go wild and
```python
class Transformer:
    def __inv__(self):
        return self.execute()
```

```python
res = ~(Ignition(engine) >>
       HeadFiles() >>
       LanguageSelector())
```

Now it starts to look like real cryptic C++!

Reply:
Nice!
Actually, we can have a 
```python
class Execute(Transformer):
    def __call__(self, head):
        return head.execute()
```

So 
```python
res = (Ignition(engine) >>
        HeadFiles() >>
        LanguageSelector() >>
        Execute())
```

I just do not like to use inverse symbol `~` for it.

Reply:
Good idea

Reply:
Who wants to implement?

Reply:
I will in another PR